jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -91,8 +58,8 @@ def vue_calculate_moment(self, *args):
         # Need transpose to align JWST mirror shape. Not sure why.
         self.moment = CCDData(analysis.moment(slab, order=n_moment).T)
 
-        label = "Moment {}: {}".format(n_moment, self._selected_data.label)
-        fname_label = self._selected_data.label.replace("[", "_").replace("]", "")
+        label = "Moment {}: {}".format(n_moment, self.dataset_selected)
```

Comment:
Oh no... I can see where this is going... type annotations! 👻 

p.s. While you are at this, might as well change `format.()` to f-string?

Reply:
Oh yeah, good call on the f-strings, didn't even register in my brain when I looked at it.

Reply:
I agree with you, and that's probably why it used to always be `selected_`.  But I thought it was worth the change to be self consistent with a prefix.  

I'll explain here just so that there is a paper trail, and we can think about adding some of this into the dev docs somewhere.  Each component/mixin adds an attribute to the plugin to access the component itself while keeping the traitlets at the plugin-level so they're accessible to observe and use in the templates.  So for dataset, there is a `plugin.dataset` which points to the underlying dataset component object.  This means that you can access the traitlets directly as `plugin.dataset_selected` but also could access them with `plugin.dataset.selected`... and that's why I went with a common "component-name-prefix" convention, even if its a bit less intuitive linguistically. 🤷

Reply:
Really I think either order of the words could have either connotation, so since there's a good reason for it this way I'm cool with it.